install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -1300,8 +1300,8 @@ data:
   clustermesh-enable-endpoint-sync: {{ .Values.clustermesh.enableEndpointSliceSynchronization | quote }}
   clustermesh-enable-mcs-api: {{ .Values.clustermesh.enableMCSAPISupport | quote }}
 
-# Extra config allows adding arbitrary properties to the cilium config.
-# By putting it at the end of the ConfigMap, it's also possible to override existing properties.
+  # Extra config allows adding arbitrary properties to the cilium config.
+  # By putting it at the end of the ConfigMap, it's also possible to override existing properties.
 {{- if .Values.extraConfig }}
```

At the very least the commit message and PR description should be updated for this PR to reflect that this is a cosmetic change and not fixing a bug. Otherwise we may mislead other users into thinking that this solves the bug, when they actually need to take action in the way they specify the helm arguments.

I noticed that some of the other examples in this file put the comment inside the if statement, which means that the comment would not be shown unless the user specifies extra config arguments:
```suggestion
{{- if .Values.extraConfig }}
  # Extra config allows adding arbitrary properties to the cilium config.
  # By putting it at the end of the ConfigMap, it's also possible to override existing properties.
```